### PR TITLE
EIP1-5853: Api spec update to include pagination while searching AEDs

### PR DESCRIPTION
**Pull request by vishalgupta-valtech**
image

### CODE REVIEW

File: src/main/resources/openapi/PrintAPIs.yaml [hunk 1]
```diff
@@ -525,6 +525,23 @@ paths:
...
+        - name: page
+          description: The page number being requested (with the first page being 1). Defaulted to 1 if not supplied
+          schema:
+            type: integer
+            minimum: 1
+          example: 1
+          in: query
+          required: true
+        - name: pageSize
+          description: The number of results to return in each page. Defaulted to 100 if not supplied
```
Comment (matt-wills-valtech) [inline src/main/resources/openapi/PrintAPIs.yaml:537]: Is this on the assumption the UI is going to be ready?

Comment (vishalgupta-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:537]: Even if UI doesn't supply page or pageSize, the controller defaults them to 1 and 100, so shouldn't break UI.

Comment (vishalgupta-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:537]: image

Comment (matt-wills-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:537]: What if there are > 100 results? I thought we were going to set it to something high, like 1000 to start wtih?

Comment (vishalgupta-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:537]: Yes sure, Could change that in next subtasks.

Comment (vishalgupta-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:537]: As per great @matt-wills-valtech there are only around 3570 anonymous electors, so chances of AEDs among some ero's applications going above are going to be less than 100, except one of two EROS which has more AEDs. 😉

Comment (matt-wills-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:537]: Perhaps the best thing is to make a decision on this tomorrow, depending on how the API & UI dev goes, so I'll approve this 👍

**Review by matt-wills-valtech — APPROVED** [review]

**Pull request merged**